espnet2/enh/layers/dnn_iva.py

```diff
@@ -0,0 +1,148 @@
+"""DNN beamformer module."""
+import logging
+from distutils.version import LooseVersion
+from typing import List, Optional, Tuple, Union
+
+import torch
+from torch.nn import functional as F
+from torch_complex.tensor import ComplexTensor
+
+from espnet2.enh.layers.complex_utils import stack, to_double, to_float
+from espnet2.enh.layers.iva import auxiva_iss
+from espnet2.enh.layers.mask_estimator import MaskEstimator
+
+is_torch_1_9_plus = LooseVersion(torch.__version__) >= LooseVersion("1.9.0")
```

Comment:
How about use this flag to stop the support for lower-version PyTorch? We can add an assertion in the initialization of `DNN_IVA` to force torch 1.9.0+ to be used.